spec

```diff
@@ -744,14 +744,20 @@ beginseg
     include "$(BUILD_DIR)/src/libu64/pad.o"
     include "$(BUILD_DIR)/src/libu64/code_800FC620.o"
     include "$(BUILD_DIR)/src/libu64/padsetup.o"
-#else
+#endif
+#if !PLATFORM_N64
     include "$(BUILD_DIR)/src/libu64/logseverity_gc.o"
     include "$(BUILD_DIR)/src/libu64/gfxprint.o"
     include "$(BUILD_DIR)/src/libu64/rcp_utils.o"
     include "$(BUILD_DIR)/src/libu64/loadfragment2_gc.o"
-#if OOT_DEBUG
+#endif
+#if DEBUG_FEATURES
+    // Because there are no N64 Debug ROMs to reference, it is impossible to know where this file
+    // would reside on an N64 Debug Build. To allow any version to compile with debug features
```

Comment:
I think it's not impossible actually, based on how libultra files are ordered in different versions, it seems like the linker puts library files in a deterministic order based on when they're first referenced. Or maybe we could reverse-engineer the linker (not that I want to do that right now)

Also it only now occurred to me that we could write this as

```
#if PLATFORM_N64
   ...
#if DEBUG_FEATURES
   // this is a guess etc
   include "$(BUILD_DIR)/src/libu64/mtxuty-cvt.o"
#endif
   ...
#else
   ...
#if DEBUG_FEATURES
   include "$(BUILD_DIR)/src/libu64/mtxuty-cvt.o"
#endif
   ...
#endif
```
which would keep the `#else` between the two libu64 versions, which might be a bit cleaner. What do you think?